lib/couchbase-orm/encrypt.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true, encoding: ASCII-8BIT
+
+module CouchbaseOrm
+    module Encrypt
+        TANKER_ENCRYPTED_PREFIX = 'tanker_encrypted_'
+
+        def encrypted_attributes(attributes)
+            attributes.clone.each do |key, value|
+                if key.to_s.starts_with?(TANKER_ENCRYPTED_PREFIX)
+                    attributes["encrypted$#{key}"] = {
+                        alg: 'CB_MOBILE_CUSTOM',
+                        ciphertext: value
+                    }
+                    attributes.delete(key)
+                end
+            end
+        end
+
+        def decrypted_attributes(attributes)
```

Comment:
Maybe emphasis in the name that it's not encrypt/decrypt but encode/decode. The attributes remain encrypted in both process, only the data format change.